.circleci/config.yml

```diff
@@ -140,97 +93,33 @@ jobs:
         steps:
             - assume-role-and-persist-workspace:
                   aws-account: $AWS_ACCOUNT_PRODUCTION
-
-    terraform-init-and-apply-to-development:
-        executor: docker-terraform
-        steps:
-            - terraform-init-then-apply:
-                  environment: "development"
-
-    terraform-init-and-apply-to-staging:
-        executor: docker-terraform
-        steps:
-            - terraform-init-then-apply:
-                  environment: "staging"
-
-    terraform-init-and-apply-to-production:
-        executor: docker-terraform
-        steps:
-            - terraform-init-then-apply:
-                  environment: "production"
 workflows:
     version: 2
     continuous-delivery-development:
-        jobs:
-            - install-dependencies
-            - lint-and-test:
-                  requires:
-                      - install-dependencies
+        jobs:                   
```

Comment:
We need to have the "  filters: branches: only: master" for each job so the workflow only gets triggered when a PR is approved and merged into master